modules/ding_user_form/js/ding_user_form.js

```diff
@@ -24,18 +24,24 @@
         }
       }
 
-      $('a.js-topbar-link-user,a.open-login', context).on('click', function(evt) {
-        evt.preventDefault();
+      function openHandler(event) {
+        event.preventDefault();
         openLogin();
-      });
+      }
 
-      $('.close-user-login', context).on('click', function(evt) {
+      function closeHandler(event) {
         var body = $('body');
 
-        evt.preventDefault();
+        event.preventDefault();
         body.removeClass('pane-login-is-open');
         body.removeClass('overlay-is-active');
-      });
+      }
+
+      $('a.js-topbar-link-user,a.open-login', context).unbind('click', openHandler);
+      $('a.js-topbar-link-user,a.open-login', context).on('click', openHandler);
```

Comment:
I understand the reasoning behind this change but I am not that keen on the implementation.

1. [`unbind()` is deprecated in jQuery 3 and has been replaced by `off()`](https://api.jquery.com/unbind/#entry-longdesc)
2. Could we use [`one()`](https://api.jquery.com/one/) instead of rebinding to ensure that handlers are only bound once?

Reply:
Changed to one() :-)